### ISSUE TITLE
Problem while compiling injector

### ISSUE THREAD

**Issue by Yatagarasu-dev** [amizu03/VAC-Bypass-Injector#1]
More than 43 errors appear when trying to compile the injector! How to fix this, and what version of Visual Studio do you have?

**Comment by amizu03** [comment]
I use VS Code, but should compile fine on VS2019. Please provide more information as to what errors you are having.

**Comment by Yatagarasu-dev** [comment]
Uploading bandicam 2021-03-16 12-10-20-050.mp4…

**Comment by Yatagarasu-dev** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/80620556/111284362-1bb12800-8651-11eb-9441-92908ec4126b.png)
![Screenshot_2](https://user-images.githubusercontent.com/80620556/111284368-1ce25500-8651-11eb-8fd5-817a063c9aca.png)
![Screenshot_3](https://user-images.githubusercontent.com/80620556/111284371-1ce25500-8651-11eb-865c-5888b968d173.png)
![Screenshot_4](https://user-images.githubusercontent.com/80620556/111284373-1ce25500-8651-11eb-90e8-b78fc591e37c.png)

**Comment by Yatagarasu-dev** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/80620556/111284432-2cfa3480-8651-11eb-9817-d248f8dfa6eb.png)
![Screenshot_3](https://user-images.githubusercontent.com/80620556/111284435-2d92cb00-8651-11eb-8223-598d92d03c3e.png)
![Screenshot_4](https://user-images.githubusercontent.com/80620556/111284437-2e2b6180-8651-11eb-8772-aa466a721182.png)

**Comment by Yatagarasu-dev** [comment]
Sorry for the errors in Russian, try to find the error code on the Internet on the Microsoft website

**Comment by amizu03** [comment]
**Help Me Understand Your Problem**
I don't mean to be rude, but It's not my job to sift through the Russian errors, and look everything up for you. It's a waste of time for me, and is something you should be doing, since you are the one asking for help. At least help me understand the problem. Don't leave everything to me.

**Compilation Instructions**
You should be able to compile by simply running the .bat file. If you want to compile it in visual studio make sure you compile with the "c++latest" language option. Also, I don't know why you created a console application project. Just create an empty project and move everything in there. Everything is functioning as is and all the code was included in the repository. Of course I can't guarantee it will compile, because I don't know specifically what you are doing. It seems you have created a separate project with completely different settings. I'm sorry I never specified any instructions. I'll make sure to fix that right now.

**Further Support**
TLDR; I can't help you with an altered version of the project since I do not have access to your source code, and it is probably a modified version of it. There is no way to read your mind. You have not provided any background information. I see you have modified version of the project, but you never disclosed this, or gave any additional information that would aid me in helping you. My discord ID (ses#1997) is attached on my profile if you need further assistance. It would make helping you a lot easier if you can add me so I can address the problem directly.

**Comment by amizu03** [comment]
No reply. Issue will be closed.
